pyproject.toml

```
@@ -0,0 +1,30 @@
+[build-system]
+requires = [
+  "setuptools>=67.8",
+]
```

Let's be explicit about the backend:

* https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
* https://packaging.python.org/en/latest/tutorials/packaging-projects/

```suggestion
[build-system]
requires = [
  "setuptools>=67.8",
]
build-backend = "setuptools.build_meta"
```